mne/viz/tests/test_3d.py

```diff
@@ -316,4 +317,33 @@ def test_snapshot_brain_montage():
     assert_raises(TypeError, snapshot_brain_montage, fig, info)
 
 
+@testing.requires_testing_data
+@requires_pysurfer
+@requires_mayavi
+def test_plot_vec_source_estimates():
+    """Test plotting of vector source estimates."""
+    sample_src = read_source_spaces(src_fname)
+
+    vertices = [s['vertno'] for s in sample_src]
+    n_verts = sum(len(v) for v in vertices)
+    n_time = 5
+    data = np.random.RandomState(0).rand(n_verts, 3, n_time)
+    stc = VectorSourceEstimate(data, vertices, 1, 1)
+    plot_vec_source_estimates(stc, 3, 'sample', subjects_dir=subjects_dir)
```

Comment:
how about just calling
stc.plot?

it will make sure stc.plot is covered by the tests.